src/Utilities.jl

```diff
@@ -191,32 +192,32 @@ struct RootRhs
     rhs
     mpm
 end
-function (rr::RootRhs)(x)
+function (rr::RootRhs)(x,p)
```

Comment:
hm commit suggestion is gone...

maybe just
```
function (rr:RootRhs)(x, p=nothing, t=nothing)
```
to achieve the old behavior? Also might be worth adding an option for `t`. Maybe `nothing` would be the better default than `0.0` since it screams a the user in case there is an explicit `t` dependency.

Bit off topic, but since the `rhs` and `mpm` fields are not typed the inner loop of `nlsolve` will be type unstable. But this might not be a problem in practice as it converges fast for small systems and for large systems the matrix multiply cost might dominate the runtime dispatch cost 🤷‍♂️